gapic-common/gapic-common.gemspec

```diff
@@ -40,6 +40,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "googleapis-common-protos-types", ">= 1.0.4", "< 2.0"
   spec.add_dependency "googleauth", "~> 0.9"
   spec.add_dependency "grpc", "~> 1.25"
+  spec.add_dependency "faraday", "~> 1.3.0"
```

Comment:
Are you sure you want to limit it to 1.3.x?

Reply:
fixed